autode/opt/optimisers/crfo.py

```diff
@@ -6,38 +6,67 @@
 [2] J. Baker, J. Comput. Chem., 13, 240 Ž1992
 """
 import numpy as np
-from typing import Union
+from typing import Union, Optional, List, TYPE_CHECKING
 
 from autode.log import logger
 from autode.values import GradientRMS, Distance
 from autode.opt.coordinates import CartesianCoordinates, DICWithConstraints
 from autode.opt.coordinates.internals import AnyPIC
 from autode.opt.optimisers.rfo import RFOptimiser
+from autode.exceptions import OptimiserStepError
 from autode.opt.optimisers.hessian_update import (
     BFGSDampedUpdate,
     BFGSSR1Update,
 )
 
+if TYPE_CHECKING:
+    from autode.opt.coordinates.primitives import Primitive
+
+# max and min bounds for the trust radius
+_max_trust = 0.2
+_min_trust = 0.01
```

Comment:
(probably not consistent, but we should probably do it)
these names should probably be uppercase https://peps.python.org/pep-0008/#constants